docs/aws-professional/eks-to-aks/node-pools-content.md

```diff
@@ -271,7 +271,7 @@ In the **Virtual node usage** column:
 | `kubernetes.azure.com/vnet` | `<nodepool virtual network name>` | `vnetName` | Virtual node virtual network |
 | `kubernetes.azure.com/ppg` | `<nodepool ppg name>` | `ppgName` | N/A |
 | `kubernetes.azure.com/encrypted-set` | `<nodepool encrypted-set name>` | `encrypted-set-name` | N/A |
-| `kubernetes.azure.com/accelerator` | `<accelerator>` | `Nvidia` | N/A |
+| `kubernetes.azure.com/accelerator` | `<accelerator>` | `NVIDIA` | N/A |
```

Comment:
I think this k8s label is actually all lowercase when it's emitted from a `kubectl describe node` operation

```suggestion
| `kubernetes.azure.com/accelerator` | `<accelerator>` | `nvidia` | N/A |
```